raiden/accounts.py

```
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import json
+import getpass
+
+from pyethapp.accounts import Account
+
+
+def find_datadir():
+    home = os.path.expanduser("~")
```

Fail fast, fail early. Will add a check.